compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -298,7 +380,7 @@ internal class RootNodeOwner(
     private inner class OwnerImpl(
         layoutDirection: LayoutDirection,
         override val coroutineContext: CoroutineContext,
-    ) : Owner {
+    ) : Owner, FocusChangeNotifier {
```

Comment:
`Owner` is the right entry point for this logic, but it's better to extract it into a separate object

Reply:
I'm using `OwnerImpl` object to get to the required implementation of `FocusChangeNotifier` in [DelegatableNode.skiko.kt](https://github.com/JetBrains/compose-multiplatform-core/pull/1523/files#diff-428f533465f46211104d4a2f85b15205151a1c4dc55090cda55b3654bf9fdf96).
Not sure it can be done easily with a separate object. What was you idea here?

Reply:
Having internal property to that for that is OK, but that new public-stable `DelegatableNode.bringFocusIntoViewParent` API requires more attention 👀

Reply:
As discussed, `@InternalComposeUiApi` attribute added. Considering to move `BringIntoViewParent` logic to UI module.

Reply:
The initial point is not solved - please extract this to a separate object from `Owner`